### PR TITLE
Create bodymass.py

### PR DESCRIPTION
**Pull request by erent8**
thanks for project.

### CODE REVIEW

**Comment by erent8** [comment]
thanks for project.

File: bodymass.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+kilo = float (input("kilonuzu giriniz(örnek: 84.9): "))
```
Comment (NitkarshChourasia) [inline bodymass.py:1]: Encapsulate it in function.

File: bodymass.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+boy = float (input("Boyunuzu m cinsinden giriniz: "))
+
+vki = (kilo / (boy**2))
+
+if vki < 18.5:
+    print(f"vucut kitle indeksiniz:  {vki} zayıfsınız.")
+elif vki < 25:
+    print (f"vucut kitle indeksiniz: {vki} normalsiniz.")
```
Comment (NitkarshChourasia) [inline bodymass.py:9]: If using so many if-elif-else statements, then use match-case, which is much more better.

File: bodymass.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+elif vki < 30:
+    print (f"vucut kitle indeksiniz: {vki} fazla kilolusunuz.")
+elif vki < 35:
+    print (f"vucut kitle indeksiniz: {vki} 1. derece obezsiniz")
+elif vki < 40:
```
Comment (NitkarshChourasia) [inline bodymass.py:14]: Don't use **Magic Numbers.**

File: bodymass.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    print (f"vucut kitle indeksiniz: {vki} 2.derece obezsiniz.")
+elif vki >40:
+    print (f"vucut kitle indeksiniz: {vki} 3.derece obezsiniz.")
+else:
+    print("Yanlış değer girdiniz.")
```
Comment (NitkarshChourasia) [inline bodymass.py:19]: use
```python
if __name__ == "__main__":
        return main()
```

File: bodymass.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+kilo = float (input("kilonuzu giriniz(örnek: 84.9): "))
+boy = float (input("Boyunuzu m cinsinden giriniz: "))
+
+vki = (kilo / (boy**2))
```
Comment (NitkarshChourasia) [inline bodymass.py:4]: Please use a standard reading language.
# Please use **ENGLISH** language for writing programs with python, obviously.

**Review by NitkarshChourasia — CHANGES_REQUESTED** [review]

**Pull request merged**